app/javascript/components/top-navbar/notifications.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from 'patternfly-react';
+import { connect } from 'react-redux';
+
+const Notifications = ({
+  unreadCount,
+}) => (
+  <li className="dropdown">
+    <a
+      id="notifications-btn"
+      className="nav-item-iconic drawer-pf-trigger-icon"
+      title={`${unreadCount} ${__('unread notifications')}`}
+      onClick={(event) => {
+        sendDataWithRx({ type: 'toggleNotificationsList' });
+        event.preventDefault();
+      }}
+    >
+      <Icon type="fa" name="bell" />
+      <span className="badge badge-pf-bordered">{unreadCount > 0 ? ' ' : ''}</span>
```

Comment:
Just want to add note here for anyone that might think this is strange. This is actually how you have to trigger the blue circle above the icon. It checks "empty" css selector to make the decision to hide/show the circle.